src/python/install.sh

```diff
@@ -386,11 +390,14 @@ if [ "${PYTHON_VERSION}" != "none" ]; then
         IFS=$OLDIFS
     fi
 
-    chown -R "${USERNAME}:python" "${PYTHON_INSTALL_PATH}"
-    chmod -R g+r+w "${PYTHON_INSTALL_PATH}"
-    find "${PYTHON_INSTALL_PATH}" -type d | xargs -n 1 chmod g+s
+    if [ ${PYTHON_VERSION} != "os-provided" ] && [ ${PYTHON_VERSION} != "system" ]; then
+        updaterc "if [[ \"\${PATH}\" != *\"${CURRENT_PATH}/bin\"* ]]; then export PATH=${CURRENT_PATH}/bin:\${PATH}; fi"
```

Comment:
We likely don't need to `updaterc` (and should update other scripts that use it) if the scripts are used as features, since the `devcontainer-feature.json` already defines these as `containerEnv`.

We need to make sure we are standardizing our scripts to not use be setting variables like $PATH  in multiple ways

Reply:
Can be handled in another PR since that is irrelevant to this change